spec/app_board_view_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "spec_helper"
+
+RSpec.describe "Board view", type: :feature do
+  before do
+    Capybara.app = Sinatra::Application
+    stub_const("CACHE_EXPIRY_TIMEOUT", 0)
+  end
+
+  context "get '/board/dxw'" do
```

Comment:
These tests work, but one thing to be aware of is that they are testing the current state of those boards and if (for example) the Leeds board turns off the clock then the tests will fail even though the application is working as intended.

The way around this would be to have a 'test boards' configuration file which is loaded for testing (maybe via a `BOARDS_CONFIG` environment variable), but this is probably only really worth doing if/when we run into the problem!

Reply:
Good point. I didn't consider that scenario. I agree with what you said though, we can cross that bridge when we come to it